validation.go

```diff
@@ -67,7 +87,13 @@ type ValidationLessThan struct {
 }
 
 func (v ValidationLessThan) validate(value string) error {
-	// TBD
+	val, err := strconv.ParseFloat(value, 64)
+	if err != nil {
+		return fmt.Errorf("%q value should be a number: %w", value, ErrInvalidParamValue)
+	}
+	if !(val < v.Value) {
+		return fmt.Errorf("%q should be less than %.4f: %w", value, v.Value, ErrLessThanValidationFail)
+	}
```

Comment:
Shouldn't we have different GT and LT validations for ints and floats, since the respective value ranges are different?

Reply:
@maha-hajja What I meant is that in this code we have in `ValidationLessThan` works on floats but since integers and floats have different value ranges, that won't always be correct. For example, what if the value cannot fit into an integer (e.g. too big)? The code above would pass (because the value can fit into a float). But, as I was typing this I realized that we have a type check which comes before this, so you can ignore this!